test-collateral/generate-policy/src/main.rs

```diff
@@ -0,0 +1,524 @@
+                .value_name("URL")
+                .help("URL of the Tabasco server.")
+                .required(true),
+        )
+        .arg(
+            Arg::with_name("output-policy-file")
+                .short("o")
+                .long("output-policy-file")
+                .value_name("FILE")
+                .help("Filename of the generated policy file.")
+                .default_value(DEFAULT_OUTPUT_FILENAME)
+                .required(true),
+        )
+        .arg(
+            Arg::with_name("certificate-lifetime-in-hours")
+                .short("l")
+                .long("certificate-lifetime-in-hours")
+                .value_name("LIFETIME")
+                .help(
+                    "Describes the expiry lifetime of the server certificate, measured in \
+hours.",
+                )
+                .default_value(&default_expiry)
+                .required(true),
+        )
+        .arg(
+            Arg::with_name("data-provision-order")
+                .short("f")
+                .long("data-provision-order")
+                .value_name("ORDER")
+                .help("Specifies the data provisioning order.")
+                .required(false),
+        )
+        .arg(
+            Arg::with_name("stream-provision-order")
+                .short("a")
+                .long("stream-provision-order")
+                .value_name("ORDER")
+                .help("Specifies the streaming provisioning order.")
+                .required(false),
+        )
+        .arg(
+            Arg::with_name("binary")
+                .short("b")
+                .long("binary")
+                .value_name("FILE")
+                .help("Specifies the filename of the WASM binary to use for the computation.")
+                .required(true),
+        )
+        .arg(
+            Arg::with_name("debug")
+                .short("d")
+                .long("debug")
+                .help(
+                    "Specifies whether the Veracruz trusted runtime should allow debugging \
+information to be produced by the executing WASM binary.",
+                )
+                .required(true)
+                .default_value(&default_debug),
+        )
+        .arg(
+            Arg::with_name("execution-strategy")
+                .short("x")
+                .long("execution-strategy")
+                .help(
+                    "Specifies whether to use interpretation or JIT execution for the WASM \
+binary.",
+                )
+                .required(true)
+                .default_value(DEFAULT_EXECUTION_STRATEGY),
+        )
+        .get_matches();
+
+    info!("Parsed command line.");
+
+    let mut arguments = Arguments::new();
+
+    if let Some(certificates) = matches.values_of("certificate") {
+        arguments.certificates = certificates.map(|s| String::from(s)).collect();
+    } else {
+        eprintln!("No certificates were passed as command line parameters.");
+        exit(-1);
```

Comment:
Out of curiosity, why exit(1) vs exit(-1) in some places?

Reply:
Just a mistake, they should all be `exit(1)`.  I will normalise them.